app/src/main/res/layout/fragment_signin.xml

```diff
@@ -8,42 +8,79 @@
     >
 
     <TextView
-        android:id="@+id/hiAllTextView"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:text="Welcome"
+        app:layout_constraintBottom_toTopOf="@id/hiAllTextView"
+        app:layout_constraintStart_toStartOf="@id/hiAllTextView"
+        tools:ignore="HardcodedText"
+        style="@style/TextAppearance.MaterialComponents.Body1"
```

Comment:
but it's strange right? I mean, It's strange that an app uses just material, in the usual case you need to change something.

Reply:
But it's a template, so maybe it's fine

Reply:
i mean it shows the right style, i can made the style for this but i think that maybe it would be a little bit of overhead for a sample app